spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/core/LoadBalancerServiceInstanceCookieTransformer.java

```diff
@@ -39,16 +40,25 @@
 
 	private final LoadBalancerProperties.StickySession stickySessionProperties;
 
+	private LoadBalancerPropertiesFactory propertiesFactory;
+
+	@Deprecated
```

Comment:
Please add javadoc with `@deprecated` tag indicating the suggested replacement.

Reply:
Added